test.disabled/ejabberd_tests/tests/push_SUITE.erl

```diff
@@ -0,0 +1,715 @@
+-module(push_SUITE).
+-compile(export_all).
+-include_lib("escalus/include/escalus.hrl").
+-include_lib("common_test/include/ct.hrl").
+-include_lib("eunit/include/eunit.hrl").
+-include_lib("escalus/include/escalus_xmlns.hrl").
+-include_lib("exml/include/exml.hrl").
+
+-define(NS_PUSH,                <<"urn:xmpp:push:0">>).
+-define(NS_XDATA,               <<"jabber:x:data">>).
+-define(NS_PUBSUB_PUB_OPTIONS,  <<"http://jabber.org/protocol/pubsub#publish-options">>).
+-define(PUSH_FORM_TYPE,         <<"urn:xmpp:push:summary">>).
+-define(MUCHOST,                <<"muclight.localhost">>).
```

Comment:
We should avoid hardcoding the main XMPP domain served by the target server. You can easily read it from tests configuration via:
```erlang
ct:get_config({hosts, mim, domain})
```